generators/app/index.js

```diff
@@ -66,14 +67,12 @@ var Generator = generators.Base.extend({
 
     // Conditional prompts
     _.find(prompts, {name: 'ui'}).when = function(props) {
```

Comment:
It would be better to remove the conditional then, as the question is now asked all the time